initialScreen.ts

```diff
@@ -43,18 +43,29 @@ function renderStartScreen() {
 	)
 	document.body.textContent = ''
 	document.body.appendChild(templateInitialScreen)
+	countToWin = 0
+	seconds = 0
+	minutes = 0
 
-	function setDifficulty(event) {
-		const target = event.target
+
+	function setDifficulty(event: Event) {
+
+		const target = event.target as HTMLElement
+
+		if (target === null) throw new Error('не задана сложность игры')
 
 		if (!target.classList.contains('start-screen__difficulty')) return
 
+		if (difficultyButtons === null) throw new Error('кнопки выбора сложности не создались')
```

Comment:
Ты делаешь throw, а ты ловишь эти ошибки в коде выше? Обрабатываешь?

Reply:
да, по интерпретатору эта строка идёт впереди, но если не сделать еще одну проверку на null внутри функции, тайпскрипт ругается 'TS2531: Object is possibly 'null'.'

```suggestion
		if (difficultyButtons === null || startButton === null) throw new Error('кнопки выбора сложности или кнопка старта не создались')
```

Reply:
Я про то, ловишь ли ты эту ошибку try catch блоком где-то? Что произойдет, если выскочит данная ошибка?